### DISCUSSION TITLE
[Bug] 👍 reaction missing if mobile screen too narrow

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#49571] [Discussions]
### Select Topic Area

Bug

### Body

When accessing reactions from mobile, 👍 is ommitted if screen is not wide enough:

![Screenshot_2023-03-08-16-30-10-11_3aea4af51f236e4932235fdada7d1643](https://user-images.githubusercontent.com/10906554/223892366-0e247a50-59ef-4978-bf94-3ddf3e09a820.jpg)

This leaves the only closest option of ❤️ but that may convey too strong a feeling (or mislead recipient into believing a romantic motive is involved)

### The solution

Move the bar align w/ right edge of viewport

**Comment by LindseyB** [comment]
This is a bug. Thanks for the report.
